rdrf/rdrf/forms/dynamic/field_lookup.py

```diff
@@ -356,7 +356,15 @@ class DatatypeEWidgetSomeDataTypeName
                     widget = None
 
                 if self.cde.allow_multiple:
-                    widget = widget or CheckboxSelectMultiple
+                    if self.cde.randomise:
+                        widget = widget or CheckboxSelectMultiple(
```

Comment:
I'm not sure we want to allow overwriting the `widget` if the CDE is set to `randomise`. 
Can't we just always set the widget to the read-only CheckboxSelectMultiple?

Also can you please add `disabled` to the attributes to give a visual indication that the checkbox is disabled?


It might be cleaner if you just created a `ReadOnlyChecboxSelectMultiple` that is analogous to the `ReadOnlySelect` we already have.
Then you could have the styling in there and could also gray the label to make it obvious that the checkbox's value can't be changed.

Reply:
Added the ReadOnlyChecboxSelectMultiple widget. Can't made the checkbox disabled as in that case the value won't be submitted to the backend.